copier.yaml

```diff
@@ -1,46 +1,69 @@
 _subdirectory: "tmpl"
 _templates_suffix: ".j2"
 
+# author_name:
 full_name:
   help: "Your full name"
   default: "Monty Python"
   type: str
 
+# author_email:
 email:
   help: "Your email address"
   default: "{{ full_name.lower().replace(' ', '.') }}@meteoswiss.ch"
   type: str
 
-github_username:
-  help: "Your user name on github"
-  default: "{{ full_name.lower().replace(' ', '_') }}"
-  type: str
-
 project_name:
-  help: "Human-friendly project name"
+  help: "Human-friendly project name used in READMEs etc."
   default: "Flying Circus"
   type: str
 
 project_slug:
-  help: "URL-friendly project name (no spaces)"
+  help: "URL-friendly project name used in github/gitlab URL etc. (no spaces)"
   default: "{{ project_name.lower().replace(\"'\", '').replace(' ', '-') }}"
   type: str
 
+# project_package:
 module_name:
   help: "Python module name (no spaces or dashes)"
   default: "{{ project_slug.replace('-', '_') }}"
   type: str
 
+# project_description:
 project_short_description:
-  help: "Your project in one line"
+  help: "Describe your project in one line"
   default: "{{ full_name }}'s {{ project_name }}"
   type: str
 
+project_keywords:
+  help: "Keywords that describe your project (comma-separated)"
+  default: "{{ project_name }}"
+  type: str
+
+# project_git_user:
+github_username:
+  help: "Name of github/gitlab user hosting the project"
+  # default: "MeteoSwiss-APN"
```

Comment:
remove

Reply:
Remove what exactly?

Reply:
I was originally referring to the old default, which was commented out. But Given the other answer, I'll set the default here back to MeteoSwiss-APN and delete the other default.